_includes/integrations/react-tech-ref.md

```diff
@@ -284,40 +484,178 @@ Use the `tinymceScriptSrc` prop to specify an external version of TinyMCE to laz
 ```
 
 #### `value`
-This property allows the editor to be used as a controlled component by setting the `value` property and using the `onEditorChange` event.
+
+The HTML content of the editor when operating as a controlled component. 
+
+When this property is different to the current editor content, the editor content
+will be changed to match (within 200 milliseconds) and an undo level will be created.
+When the editor content changes by this mechanism an attempt will be made to
+retain the selection, however if the previous selection does not exist in the new
+content the cursor will be returned to the start of the document.
+
+This property allows the editor to be used as a controlled component by setting 
+the `value` property and using the `onEditorChange` event to update the `value`.
 
 For more information, see: [Using the {{site.productname}} React component as a controlled component](#usingthetinymcereactcomponentasacontrolledcomponent).
 
 **Type:** String
 
-### Using the TinyMCE React component as a controlled component
+### Using the {{site.productname}} React component as a uncontrolled component
+
+The {{site.productname}} React component is best used as a uncontrolled component
+as it allows the editor to perform well on larger documents.
 
-To use the editor as a [controlled component](https://reactjs.org/docs/forms.html#controlled-components), use the `onEditorChange` event instead of the `onChange` event, such as:
+When using the editor in the uncontrolled mode you must avoid using the `value`
+and `onEditorChange` props. Instead, get the content of the editor directly
+when it is needed. This is easiest done by storing the editor reference when the
+`onInit` event handler is called.
+
+If you need to provide visual feedback to the user when there is content ready 
+to be saved you can use the `onDirty` event handler combined with clearing
+the dirty state when saving the editor content.
+
+##### Example: Functional uncontrolled component with save button and dirty state
+
+```jsx
+function MyComponent({initialValue}) {
+  const edRef = useRef(null);
```

Comment:
`editorRef` here as well maybe just for consistency?

Reply:
done